docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1084 @@
+        -    A command is valid only when the command language is the same as the current language. The current language can be changed by changing the application or display language on the device.
+        -    You can get a notification of the language change in a callback. If the display language is changed to a non-supported one, the voice control service language changes to the first language supported by voice control engine.
+    -   Supported language
+        -    You can retrieve a list of supported languages to check whether the language that you want is supported.
+
+
+To use the voice control manager, follow these steps:
+
+1.  Create a handle and [register event handlers](#callback).
+
+    The initialization allows the voice control to distinguish your application from any other application that also uses voice control. Only one voice control manager instance can work on the device. Therefore, if other application including voice control manager exists, your application cannot work properly on one same device.
+
+    The registered callbacks can be notified of the changes in service status, current language, recognition results, and errors.
+
+2.  Set commands.
+
+    You can create a command list, and add or remove individual commands in the list. While creating an individual command, set the command text and type for each command handle. When all the commands are created and added to the command list, set the command list to the voice control manager for recognition.
+
+3.  Prepare the voice control manager.
+
+    The preparation connects the voice control service for background work, such as recording and recognizing the user voice.
+
+    When the application initializes and prepares the voice control manager, the voice control service is invoked and connected for the background work. The service and the application communicate as server and client.
+
+4.  Get the recognized command result and ASR.
+
+    The recognized command result and ASR are sent through a registered callback.
+    You can receive the command matching result that the user uttered.
+    And if the registered command is duplicated or the user speaks multiple commands, the recognition result can contain multiple results. In this case, you can select one or reject in the `VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate()` callbacks
+
+5.  When no longer needed, unset the voice control manager.
+
+    You must disconnect the voice control service and deinitialize the voice control manager using the `Unprepare()` and `Deinitialize()` methods of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class.
+
+The following figure illustrates the voice control manager life-cycle:
+
+**Figure: Voice control manager life-cycle**
+
+![Voice control manager life-cycle](./media/csapi_vc_manager.png)
+
+The user controls the voice control manager service state. In a general scenario:
```

Comment:
@rishu-samsung According to your suggestion, this sentence would become

You control the voice control manager service state. In a general scenario:

How about below rather than above?

In a general scenario, voice control manager service state is flows :

This is just a suggestion, and it needs the sentence checking